src/js/angular/import/templates/filesTable.html

```diff
@@ -63,10 +63,10 @@ <h4>Last import settings</h4>
         <table class="table table-sm m-0" ng-init="settings = filterSettings(file.name)">
             <tr ng-repeat="setting in settings">
                 <td><strong>{{setting[0]}}</strong></td>
-                <td class="break-word" ng-show="setting[0] == 'parserSettings'">
+                <td ng-show="setting[0] == 'parserSettings' || setting[0] == 'requestIdHeadersToForward'">
```

Comment:
I wonder where this additional check came from `|| setting[0] == 'requestIdHeadersToForward'` and why it was needed?

Reply:
Left a comment in the issue.
The `requestIdHeadersToForward`'s value is a JSON object but was not displayed in a `pre` tag.